src/app-layout/__integ__/app-layout-toolbar-tooltips.test.ts

```diff
@@ -0,0 +1,361 @@
+      })
+    );
+
+    test(
+      'Removes tooltip from drawer trigger on escape key press after showing from keyboard event',
+      setupTest({ theme, size }, async page => {
+        await expect(page.getElementsCount(`.${tooltipStyles.root}`)).resolves.toBe(0);
+        await expect(page.isExisting(`.${appliedThemeStyles[drawersTriggerContainerClassKey]}`)).resolves.toBeTruthy();
+        await page.click(firstDrawerTriggerSelector);
+        //close drawer via keys
+        await page.keys(size === 'desktop' ? ['Tab', 'Enter'] : ['Enter']); //focus element is resize in desktop
+        await expect(page.isFocused(firstDrawerTriggerSelector)).resolves.toBeTruthy();
+        await expect(page.isExisting(`.${tooltipStyles.root}`)).resolves.toBe(false);
+        //navigate away from drawer triggers
+        if (theme === 'visual-refresh-toolbar') {
+          await page.keys(['Shift', 'Tab', 'Null']); // to split panel drawer
+        }
+        await page.keys(['Shift', 'Tab', 'Null']); // to last breadcrumb - be aware if breadcrumb has tooltip
+        await expect(page.isExisting(`.${tooltipStyles.root}`)).resolves.toBe(false);
+        //navigate back to drawer trigger
+        await page.keys(theme === 'visual-refresh-toolbar' ? ['Tab', 'Tab'] : ['Tab']);
+        await expect(page.isFocused(firstDrawerTriggerSelector)).resolves.toBeTruthy();
+        await expect(page.getElementsCount(`.${tooltipStyles.root}`)).resolves.toBe(1);
+        await page.keys('Escape');
+        await expect(page.isExisting(`.${tooltipStyles.root}`)).resolves.toBe(false);
+      })
+    );
+
+    test(
+      'Shows tooltip correctly on drawer trigger for keyboard (tab) interactions after drawer actions',
+      setupTest({ theme, size }, async page => {
+        await expect(page.getElementsCount(`.${tooltipStyles.root}`)).resolves.toBe(0);
+        await expect(page.isExisting(`.${appliedThemeStyles[drawersTriggerContainerClassKey]}`)).resolves.toBeTruthy();
+        await page.click(firstDrawerTriggerSelector);
+        await expect(page.isExisting(`.${tooltipStyles.root}`)).resolves.toBe(false);
+        //close drawer via keys
+        await page.keys(size === 'desktop' ? ['Tab', 'Enter'] : ['Enter']); //first focus element is resize in desktop
+        await expect(page.isFocused(firstDrawerTriggerSelector)).resolves.toBeTruthy();
+        await expect(page.isExisting(`.${tooltipStyles.root}`)).resolves.toBe(false);
+        await page.keys(['Tab']);
+        await expect(page.isFocused(secondDrawerTriggerSelector)).resolves.toBeTruthy();
+        await expect(page.getElementsCount(`.${tooltipStyles.root}`)).resolves.toBe(1);
+        await page.keys(['Shift', 'Tab', 'Null']);
+        await expect(page.isFocused(firstDrawerTriggerSelector)).resolves.toBeTruthy();
+        await expect(page.getElementsCount(`.${tooltipStyles.root}`)).resolves.toBe(1);
+      })
+    );
+
+    testIf(theme === 'visual-refresh-toolbar')(
```

Comment:
Minor: by the `describe.each` above, `theme` can only be `visual-refresh-toolbar`. Same for the next `testIf` blocks.

Reply:
there is a followup to add visual-refresh to this as it was in previous commits.

Reply:
removed the code and added a comment for when visual refresh is returned